docs/client.rst

```diff
@@ -298,6 +389,11 @@ Class                                    Purpose
 ======================================== =======================================
 :class:`~pywbem.TestClientRecorder`      Generate test cases for the
                                          `test_client` unit test module.
+
+:class:`~pywbem.LogOperationRecorder`    Generate python logs for
```

Comment:
Not sure what "python logs" are. How about just "logs"?

Reply:
Changed to using the Python logging facility.